server/candlepin.spec.tmpl

```diff
@@ -170,6 +172,7 @@ for selinuxvariant in %{selinux_variants}
 do
   /usr/sbin/semodule -s $selinuxvariant -i %{_datadir}/selinux/$selinuxvariant/%{modulename}.pp &> /dev/null || :
 done
+/usr/sbin/semanage port -a -t candlepin_activemq_port_t -p tcp 61613 &> /dev/null || :
```

Comment:
Do we want to define the port as a variable somewhere so it's not hard coded? Is that even a thing we can do?

Reply:
Given that this is in the spec file, I don't know where/how you would create a variable for it, except the spec file itself, but it would be only for readability/aesthetic reasons, not functional.

Reply:
Oh yeah... missed that. It looks like bash scripting; though I can't quite figure out exactly what that double pipe is doing at the end. Is that just to suppress any error output from the command?

Reply:
It is running bash commands, since this is inside a `#raw` block: https://stackoverflow.com/questions/16357755/meaning-of-colon-in-bash-after-a-double-pipe#comment23436601_16357803